packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -204,3 +204,13 @@ class RegisteredServiceExtensionButton {
     }
   }
 }
+
+bool shouldHideTab(String key) {
```

Comment:
maybe move the core logic for this out of src/ui 
as it is more framework business logic than ui specific.

Reply:
Where do you think should be best? It's using `dart:html` to just check if the tab is on the querystring, so I wasn't sure if we wanted to have it too far down.